packages/core/src/sync-store/postgres/store.ts

```diff
@@ -824,82 +814,65 @@ export class PostgresSyncStore implements SyncStore {
     fromCheckpoint,
     toCheckpoint,
     limit,
-    logFilters = [],
-    factories = [],
+    logFilters = undefined,
+    factories = undefined,
   }: {
     fromCheckpoint: Checkpoint;
     toCheckpoint: Checkpoint;
     limit: number;
-    logFilters?: {
-      id: string;
-      chainId: number;
-      criteria: LogFilterCriteria;
-      fromBlock?: number;
-      toBlock?: number;
-      includeEventSelectors?: Hex[];
-    }[];
-    factories?: {
-      id: string;
-      chainId: number;
-      criteria: FactoryCriteria;
-      fromBlock?: number;
-      toBlock?: number;
-      includeEventSelectors?: Hex[];
-    }[];
-  }) {
-    const start = performance.now();
+  } & (
+    | {
+        logFilters: {
+          id: string;
+          chainId: number;
+          criteria: LogFilterCriteria;
+          fromBlock?: number;
+          toBlock?: number;
+          includeEventSelector: Hex;
```

Comment:
I noticed that we're now always including the event selector in the SQL filter. Maybe we should rename this?